ci/download-release-artifacts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -ex
+
+IPFS_COMPANION_VERSION=${IPFS_COMPANION_VERSION:-$(jq -r '.version' ./add-on/manifest.common.json)}
+
+id="$(curl --retry 5 --no-progress-meter "https://api.github.com/repos/ipfs/ipfs-companion/releases/tags/v$IPFS_COMPANION_VERSION" | jq '.id')"
+assets="$(curl --retry 5 --no-progress-meter --location "https://api.github.com/repos/ipfs/ipfs-companion/releases/$id/assets" | jq -r '.[].name')"
+
+mkdir build
+
+for asset in $assets; do
+  curl --retry 5 --no-progress-meter --location --output "build/$asset" "https://github.com/ipfs/ipfs-companion/releases/download/v$IPFS_COMPANION_VERSION/$asset"
+done
```

Comment:
The problem with using an action for this is that it cannot be executed locally. I found it useful to be able to call `ci/download-release-artifacts.sh` and work with released artifacts.

Reply:
I didn't but that's because I wanted to use a version that's known to work so that I can work on the test without wondering if it's something in the building process that's affecting my results. It might not be the most common workflow though.

Reply:
I don't think this script would be called locally unless explicitly wanting to test the functionality of downloading the assets from github @whizzzkid. It should only be called by `e2e.yml` when `ipfs-companion-version` is not an empty string, or when you're wanting to validate, locally, that the script pulls from github appropriately.